src/app/components/Promo/index.jsx

```diff
@@ -36,15 +37,18 @@ const Left = styled.div`
 `;
 const Right = styled.div`
   @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    padding-left: 0.5rem;
+    ${({ dir }) =>
+      dir === 'ltr'
+        ? `padding-left: ${GEL_SPACING}`
+        : `padding-right: ${GEL_SPACING};`}
```

Comment:
I think `padding-inline-start` would work - it's equivalent to padding-left on LTR and padding-right on RTL - https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline-start

```suggestion
    padding-inline-start: ${GEL_SPACING};
```

Reply:
Cool! Didn't know this was a thing